runtime/src/bank.rs

```diff
@@ -11625,87 +11672,91 @@ pub(crate) mod tests {
 
     #[test]
     fn test_store_scan_consistency_unrooted() {
-        test_store_scan_consistency(
-            |bank0, bank_to_scan_sender, pubkeys_to_modify, program_id, starting_lamports| {
-                let mut current_major_fork_bank = bank0;
-                loop {
-                    let mut current_minor_fork_bank = current_major_fork_bank.clone();
-                    let num_new_banks = 2;
-                    let lamports = current_minor_fork_bank.slot() + starting_lamports + 1;
-                    // Modify banks on the two banks on the minor fork
-                    for pubkeys_to_modify in &pubkeys_to_modify
-                        .iter()
-                        .chunks(pubkeys_to_modify.len() / num_new_banks)
-                    {
-                        current_minor_fork_bank = Arc::new(Bank::new_from_parent(
-                            &current_minor_fork_bank,
+        for accounts_db_caching_enabled in &[true] {
```

Comment:
nits: no need to loop over? or make this `&[false, true]`?

Reply:
oh oops yeah, forgot to add the `false` back, thanks!